src/Compiler/CMakeLists.txt

```diff
@@ -54,13 +54,32 @@ add_onnx_mlir_library(CompilerUtils
   MC
   )
 
+if( ACCELERATORS_TO_BUILD )
+  add_dependencies(CompilerUtils NNPAAccel)
+else ()
+  add_dependencies(CompilerUtils NoAccel)
+endif(ACCELERATORS_TO_BUILD)
+
+
+if( ACCELERATORS_TO_BUILD )
```

Comment:
Same comments about spacing here. I'm sure I have other comments on the cmake files, but I'll give you a chance to clean them up a bit more first.